_extras/guide.md

```diff
@@ -33,6 +33,20 @@ If you've written up a diagram of the data analysis pipeline (raw data ->
 clean data -> import and analyze -> results -> visualization), it can be
 helpful to identify that you're now somewhere between clean data and analysis.
 
+## Common Difficult Concepts
+
+* Making the leap from a research question to a query (seen in some of the challenges)
+* Data import
+    * Not necessarily a concept, but we always have at least a handful of people that 
+      struggle with the table import step -- especially in the virtual context with window changing
+    * Data type options in SQLite (Integer, text, blog, real, numeric) when importing 
+      from CSV. (Maybe have a table of SQLite date types in the student mateiral).
```

Comment:
It seems my previous suggestion was only a partial improvement :laughing: 
```suggestion
      from CSV. (Maybe have a table of SQLite date types in the student material).
```

Reply:
ha! and I thought I read it carefully. 🤪